include/sequence.h

```diff
@@ -3,132 +3,131 @@
-#define NA_BGM_PIANO_SESSION 0x63           // Piano Solo "DEMO_SESSION"
-#define NA_BGM_INDIGO_GO_SESSION 0x64       // The Indigo-Go's Rehearsal "ALLPART_SESSION"
-#define NA_BGM_SNOWHEAD_TEMPLE 0x65         // Snowhead Temple "SILVER_DUNGEON"
-#define NA_BGM_GREAT_BAY_TEMPLE 0x66        // Great Bay Temple "DEEPBLUE_DUNGEON"
-#define NA_BGM_NEW_WAVE_SAXOPHONE 0x67      // "DEMO_TIDE_SAX"
-#define NA_BGM_NEW_WAVE_VOCAL 0x68          // "DEMO_TIDE_VOCAL"
-#define NA_BGM_MAJORAS_WRATH 0x69           // Majora's Wrath "MUJURA_1"
-#define NA_BGM_MAJORAS_INCARNATION 0x6A     // Majora's Incarnation "MUJURA_2"
-#define NA_BGM_MAJORAS_MASK 0x6B            // Majora's Mask Battle "MUJURA_3"
-#define NA_BGM_BASS_PLAY 0x6C               // Bass Practice "BASS_PLAY"
-#define NA_BGM_DRUMS_PLAY 0x6D              // Drums Practice "DRUMS_PLAY"
-#define NA_BGM_PIANO_PLAY 0x6E              // Piano Practice "PIANO_PLAY"
-#define NA_BGM_IKANA_CASTLE 0x6F            // Ikana Castle
-#define NA_BGM_GATHERING_GIANTS 0x70        // Calling the Four Giants
-#define NA_BGM_KAMARO_DANCE 0x71            // Kamaro's Dance "CAMARO_DANCE"
-#define NA_BGM_CREMIA_CARRIAGE 0x72         // Cremia's Carriage "DONKEY_CART"
-#define NA_BGM_KEATON_QUIZ 0x73             // Keaton "QUIZ"
-#define NA_BGM_END_CREDITS_1 0x74           // The End/Credits I "END_DEMO"
-#define NA_BGM_OPENING_LOOP 0x75            // Forest Ambush (?)		Duplicate.
-#define NA_BGM_TITLE_THEME 0x76             // Title Screen
-#define NA_BGM_DUNGEON_APPEAR 0x77          // Surfacing of Woodfall
-#define NA_BGM_WOODFALL_CLEAR 0x78          // Woodfall Clear "PURIFICATION_0"
-#define NA_BGM_SNOWHEAD_CLEAR 0x79          // Snowhead Clear "PURIFICATION_1"
-#define NA_BGM_INTO_THE_MOON 0x7B           // To the Moon
-#define NA_BGM_GOODBYE_GIANT 0x7C           //
-#define NA_BGM_TATL_AND_TAEL 0x7D           // Tatl and Tael "CHAT"
-#define NA_BGM_MOONS_DESTRUCTION 0x7E       // Moon's Destruction "MOON_VANISH"
-#define NA_BGM_NO_MUSIC 0x7F                //
-#define NA_BGM_END_CREDITS_2 0x82           // The End/Credits II "STAFFROLL2"
+#define NA_BGM_TERMINA_FIELD 0x02           // Termina Field ("FIELD")
+#define NA_BGM_CHASE 0x03                   // Forest Chase ("CHASE")
+#define NA_BGM_MAJORAS_THEME 0x04           // Majora's Theme ("STALKID")
+#define NA_BGM_CLOCK_TOWER 0x05             // The Clock Tower ("CLOCK_TOWER")
+#define NA_BGM_STONE_TOWER_TEMPLE 0x06      // Stone Tower Temple ("RUIN_DUNGEON_FRONT")
+#define NA_BGM_INV_STONE_TOWER_TEMPLE 0x07  // Stone Tower Temple Inverted ("RUIN_DUNGEON_REVERSE")
+#define NA_BGM_FAILURE_0 0x08               // Missed Event 1 ("FAILURE_0")
+#define NA_BGM_FAILURE_1 0x09               // Failed Milk Run ("FAILURE_1")
+#define NA_BGM_HAPPY_MASK_SALESMAN 0x0A     // Happy Mask Salesman ("OMENYA")
+#define NA_BGM_SONG_OF_HEALING 0x0B         // Song of Healing ("DEMO_CURE")
+#define NA_BGM_SWAMP_REGION 0x0C            // Southern Swamp and Woodfall ("DAMP_AREA")
```

Comment:
I probably would have named this after the name of the actual region instead of "swamp region", like, the pirate song didn't get named "Pirate_region"
but that's just my opinion not a big deal to me.